lib/showcase.dart

```diff
@@ -256,7 +271,7 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
             _TargetWidget(
               offset: offset,
               size: size,
-              onTap: _nextIfAny,
+              onTap: _getOnTargetTap(),
```

Comment:
I don' think it is correct way to pass a function.

Reply:
Fixed